app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
note that these attributes can have only specific values (you can find them in the requirements). So let's check it and if it's incorrect value, then set some default

Reply:
Please set the default value for attribute in case if the value is incorrect. 
(Actually the correct implementation here would be using properties to validate attribute data but we will study it later).
For now replace raising an exception to setting a default value as per previous comment.